### PR TITLE
Show callout on v1 projects to upgrade to v3

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2623** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2623" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2623?utm_source=stack-comment).

File: src/hooks/v1/RelaunchV1ViaV3Create.ts
Comment (tomquirk) [inline src/hooks/v1/RelaunchV1ViaV3Create.ts:56]: **suggestion** guess we could `map` this instead?

File: src/hooks/v1/RelaunchV1ViaV3Create.ts
Comment (tomquirk) [inline src/hooks/v1/RelaunchV1ViaV3Create.ts:127]: **question** I wonder if memoization might be slightly unsafe here. For example If some key within projectMetadata changes, the `useMemo` won't re-evaluate when it probably should

File: src/hooks/v1/RelaunchV1ViaV3Create.ts
Comment (tomquirk) [inline src/hooks/v1/RelaunchV1ViaV3Create.ts:36]: **suggestion** my bad, we actually do have a fn for this that might be helpful: `toSplit` https://github.com/jbx-protocol/juice-interface/blob/93dcc2feff8fb57632d112ab313abf0379a7af8a/src/utils/splits.ts#L16

**Review by tomquirk — APPROVED** [review]
@wraeth-eth thanks so much for finishing this off, i think this'll work well, and we can iterate on it. I left a couple comments but happy to approve.

File: src/hooks/v1/RelaunchV1ViaV3Create.ts
Comment (wraeth-eth) [reply src/hooks/v1/RelaunchV1ViaV3Create.ts:56]: map didn't work for me hence why I had to do this :( i'll try the function you suggested above, but this shit is so finicky and old I kind of just wanna keep pushing wdyt

File: src/hooks/v1/RelaunchV1ViaV3Create.ts
Comment (wraeth-eth) [reply src/hooks/v1/RelaunchV1ViaV3Create.ts:127]: maybe im missing why it wouldn't, why wouldn't it 😅

File: src/hooks/v1/RelaunchV1ViaV3Create.ts
Comment (wraeth-eth) [reply src/hooks/v1/RelaunchV1ViaV3Create.ts:36]: The formats are a lil different so im going to opt to leave this one as it should hopefully not be long lived code

File: src/hooks/v1/RelaunchV1ViaV3Create.ts
Comment (wraeth-eth) [reply src/hooks/v1/RelaunchV1ViaV3Create.ts:127]: I think due to the nature of the way that useContext works the object being returned would be a 'new' object when any of the properties change so the dep would fire the callback

Comment (wraeth-eth) [reply src/hooks/v1/RelaunchV1ViaV3Create.ts:127]: I think due to the nature of the way that useContext works the object being returned would be a 'new' object when any of the properties change so the dep would fire the callback

File: src/hooks/v1/RelaunchV1ViaV3Create.ts
Comment (tomquirk) [reply src/hooks/v1/RelaunchV1ViaV3Create.ts:56]: ye

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2623?utm_source=stack-comment).


Job Reference: bNgIkwBqux2UJLmdY4Uy

**Pull request merged**
